src/clarity-angular/datagrid/datagrid-column.ts

```diff
@@ -43,10 +44,16 @@ export class DatagridColumn extends DatagridFilterRegistrar<DatagridStringFilter
         super(filters);
         this._sortSubscription = _sort.change.subscribe(sort => {
             // We're only listening to make sure we emit an event when the column goes from sorted to unsorted
```

Comment:
Exactly, the sort action can also be performed outside this class, i.e. in the Sort provider via the public method.